build.sbt

```diff
@@ -22,7 +22,7 @@ lazy val root = (project in file("."))
       ("com.gu" %% "simple-configuration-ssm" % "1.7.0").cross(CrossVersion.for3Use2_13),
       /* Using Scala 2.13 version of identity-auth-play until a Scala 3 version has been released:
        * https://trello.com/c/5kOc41kD/4669-release-scala-3-version-of-identity-libraries */
-      ("com.gu.identity" %% "identity-auth-core" % "4.23")
+      ("com.gu.identity" %% "identity-auth-core" % "2024-03-28-09-30-SNAPSHOT")
```

Comment:
this will be replaced with the prod version of the lib when it's released.

I just  have to remember to NOT merge it like this!